### PR TITLE
REST API endpoint for watchdog

### PR DESCRIPTION
**Pull request by StykMartin**
Signed-off-by: Martin Styk <mart.styk@gmail.com>

### CODE REVIEW

File: Server/bkr/server/watchdog.py [hunk 1]
```diff
@@ -1,66 +1,51 @@
...
+
+from bkr.server.app import app
+from bkr.server.flask_util import admin_auth_required, read_json_request
+from bkr.server.model import Watchdog, Recipe, RecipeSet, Job, RecipeTask
+
+
+@app.route("/watchdogs/", methods=["GET"])
+def get_watchdogs():
+    # TODO: Format may change in future based on FE requests
+    # For now give them raw data and laugh. That's what backend heroes do
```
Comment (renanrodrigo) [inline Server/bkr/server/watchdog.py:17]: hhm rawr data

File: Server/bkr/server/watchdog.py [hunk 2]
```diff
@@ -1,66 +1,51 @@
...
+
+@app.route("/watchdogs/", methods=["POST"])
+@admin_auth_required
+def set_watchdogs():
+    """Allow admins to push watchdog times out after an outage"""
+    payload = read_json_request(request)
+    time = payload.get("time")
+
+    watchdogs = {}
+    for w in Watchdog.by_status(status=u"active"):
```
Comment (renanrodrigo) [inline Server/bkr/server/watchdog.py:46]: do we still need the unicode?

**Review by renanrodrigo — COMMENTED** [review]
https://www.imdb.com/title/tt3688736/

File: Server/bkr/server/watchdog.py (hunk 2 above)
Comment (StykMartin) [reply Server/bkr/server/watchdog.py:46]: Yeah. This is still 2/3. I don't wanna screw my execution in Python 2

Comment (renanrodrigo) [reply Server/bkr/server/watchdog.py:46]: ack

**Review by renanrodrigo — APPROVED** [review]

**Comment by StykMartin** [comment]
@renanrodrigo re-review, please. I changed the final payload. Beaker was acting more like Breaker. Failed to serialize payload.

**Review by renanrodrigo — APPROVED** [review]

**Pull request merged**